kompendium-playground/src/main/kotlin/io/bkbn/kompendium/playground/Main.kt

```diff
@@ -52,10 +53,11 @@ var featuresInstalled = false
 fun Application.configModule() {
   if (!featuresInstalled) {
     install(ContentNegotiation) {
-      jackson {
-        enable(SerializationFeature.INDENT_OUTPUT)
-        setSerializationInclusion(JsonInclude.Include.NON_NULL)
-      }
+      json()
+//      jackson {
```

Comment:
Perhaps not keep the commented code?

Reply:
yep good catch, mostly just left this to show that it works w/ kotlinx... Think I will remove this, add some tests to confirm that it works regardless of serializer and then merge it